src/pages/index.tsx

```diff
@@ -14,65 +14,241 @@ const Home: NextPageWithLayout = () => {
   const { signOut } = useClerk();
 
   return (
-    <div className="bg-lightAqua-100">
-      <div className="relative h-44 w-full">
+    <div>
+      <div className="grid_bg_container relative w-full h-[25rem] sm:h-[30rem] md:h-[35rem] lg:h-[46rem] border-green-500 border-0">
```

Comment:
Instead of putting debug class names like `grid_bg_container`, I suggest adding id attributes instead since className is more reserved for styling because of tailwind. Use `kebab-case` for them